testsuite/tests/lib-unix/unix-execvpe/has-execvpe

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# This script is related to the 'exec.ml' test.
+# It tests whether the OS implements execvpe or not.
+# It makes sense to run the tests only if execvpe is nt implemented.
+# If it is implemented, the test is skipped.
+
+if grep -q HAS_EXECVPE ${ocamlsrcdir}/byterun/caml/s.h; then exit 125; fi
```

Comment:
Is there a way to use a variable name for the exit codes, such as `$EXIT_SUCCESS`, `$EXIT_FAILURE` and `$EXIT_SKIP`? Maybe the driver could always set these variables in the environment when running scripts?